package/main.yaml

```diff
@@ -0,0 +1,66 @@
+applications:
+  - appcat
+  - crossplane
+parameters:
+  pkg.appcat:
+    componentVersion: package_appcat
```

Comment:
You'll probably want to adjust this to something that isn't a feature branch name before merging

Reply:
The problem here is a classical chicken egg problem. If I do, the golden tests will fail.
However, there is a github action that will update this automatically when doing a release.